core/src/main/java/tachyon/client/RemoteBlockInStream.java

```diff
@@ -126,6 +174,7 @@
     }
 
     mBlockInfo = mFile.getClientBlockInfo(mBlockIndex);
+    mSortedWorkers = buildSortedWorkers(mBlockInfo);
```

Comment:
Should also regenerate this when mBlockInfo gets refreshed (see `updateCurrentBuffer`)